src/Pages/WishlistPage/WishlistPage.css

```diff
@@ -1,7 +1,21 @@
-  .wishlist-head {
-      display: flex;
-      justify-content: space-between;
-      font-size: 1.2rem;
-      white-space: nowrap;
-      margin-top: 1rem;
-  }
+.wishlist-head {
+  display: flex;
+  justify-content: space-between;
+  font-size: 1.2rem;
+  white-space: nowrap;
+  margin-top: 1rem;
+}
+
+.wishlist-empty-section > h1 {
+  text-align: center;
+  margin: 20rem 20rem;
+  font-size: 4rem;
+  white-space: nowrap;
+}
+
+.wishlist-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
If you're sure of the no. of columns you want, you can also write this is `grid-template-columns: repeat(4, 1fr);`